scripts/federation_flags/download_federation_flags.py

```diff
@@ -4,14 +4,21 @@
 
 # Import this to avoid circular imports
 import plugins.manager  # noqa
-from common.i18n import _
-from common.logger import print_interactive_error
 from common.sharly_chess_config import SharlyChessConfig
 from database.sqlite.fide.fide_database import FideDatabase
 
 
 def download_federation_url(federation_id: str, flag_file: Path, flag_url) -> bool:
-    response: Response = get(flag_url, allow_redirects=True, timeout=5)
+    # Add the User-Agent header to be allowed to download from WikiPedia
+    # cf https://meta.wikimedia.org/wiki/User-Agent_policy
+    response: Response = get(
+        flag_url,
+        allow_redirects=True,
+        timeout=5,
+        headers={
+            'User-Agent': f'Sharly Chess/{SharlyChessConfig.version}',
```

Comment:
Missing the contact information (necessary for additional questions from technical staff as Wikimedia)